src/main/webapp/js/common.js

```diff
@@ -450,33 +450,37 @@ function sortByDiff(a, b) {
  */
 function getPointValue(s, ditchZero) {
     var s0 = s;
+    var baseValue = 100;
+    
     if (s0.lastIndexOf('<') !== -1) {
         s0 = s0.substring(0, s0.lastIndexOf('<'));
         s0 = s0.substring(s0.lastIndexOf('>') + 1);
     }
     
```

Comment:
The above statements seems to get the text in `<html_tag>TEXT<html_tag>`. But I look through the project and there is no place that needs to skip html tag. So I didn't write test cases for this and plan to remove it. Can some senior developer confirm this. Even if there is a place that need to skip html tag, this method is not the sophisticated way to do it.